src/app/lib/posts/types.ts

```diff
@@ -13,6 +13,14 @@ export type Post = {
   }
 }
 
+export type PostPodcast = {
+  title: string
+  duration: string
+  meta_default_description?: string
```

Comment:
rename key for what they represent in the XML

`meta_default_description -> summary` or whatever it is in the XML

Also can summary and thumbnail return **null**?